### DISCUSSION TITLE
[灌水]绳网更新后貌似还有问题

### DISCUSSION THREAD

**Discussion by JTLIYOO** [share121/inter-knot#746] [其他]
虽然更新以后图片可以加载出来了，但是帖子内看不见帖子的封面图片
![QQ20240810-160138](https://github.com/user-attachments/assets/d8902982-c080-4db1-bfe0-2aa27d33d2d5)
![QQ20240810-160152](https://github.com/user-attachments/assets/6080c15d-2d12-4baf-9e9e-e09bf8c2bbba)

**Comment by Akashic007** [comment]
一时一时的，有时候就能出现，有时候又不能出现，刷新一下试试

**Comment by JTLIYOO** [comment]
进入绳网的加载时间也感觉像是变长了，是我的问题吗

**Comment by hjejdjdj** [comment]
确实

**Comment by aaa00073** [comment]
唔，感觉有时候可以，有时候不行，我昨晚反正可以。

**Comment by ww8023ql** [comment]
可能是人多了开始卡了吧
